database/things/checked-c.pldb

```diff
@@ -1,11 +1,11 @@
 title checked-c
 appeared 2015
 type pl
-originCommunity Microsoft
-country United States 
 creators David Tarditi
 website https://www.microsoft.com/en-us/research/project/checked-c/
 reference https://www.microsoft.com/en-us/research/project/checked-c/
+country United States
+originCommunity Microsoft
```

Comment:
Might this possibly be Microsoft Research?

Reply:
[It is!](https://www.microsoft.com/en-us/research/project/checked-c/)

Reply:
👍